apps/ucp-ui/src/Institutions/ChangeInstitution/api.ts

```diff
@@ -15,7 +15,9 @@ export interface EditInstitutionParams extends CreateInstitution {
 }
 
 export interface Institution {
-  id: string;
+  institution: {
+    id: string;
+  };
```

Comment:
Why add the additional "institution" prop?

Reply:
The institution create endpoint was updated to include the nested institution in the response to match the other endpoints. So this update is necessary to accommodate that update.